### PR TITLE
Assignment rahul mishra

### PR DESCRIPTION
**Pull request by RahulMTechnocredits**

### CODE REVIEW

File: JavaAssignment/src/rahulM/Q4.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+//WAP to find area and perimeter of rectangle and cirle
+package rahulM;
+class Q4
+{   static int  l,b,h;
```
Comment (harshvegada007) [inline JavaAssignment/src/rahulM/Q4.java]: logic is correct but length might comes in the double value as well.

File: JavaAssignment/src/rahulM/Q7.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+//WAP to convert seconds to hour minute and seconds
+package rahulM;
+class Q7
```
Comment (harshvegada007) [inline JavaAssignment/src/rahulM/Q7.java]: Format

File: JavaAssignment/src/rahulM/Q3.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+package rahulM;
+class Q3
+{   static int x;
+	public static void main(String[] a)
+	{
+	 int x = Integer.parseInt(a[0]);
+	 System.out.println("Square of the number is: "+square(x) + "\nCube of the number is: "+cube(x));
+	 
+	}
+	static int square(int x)
```
Comment (krishnakTechnocretids) [inline JavaAssignment/src/rahulM/Q3.java]: Method name should be like **"squareOperation."**

File: JavaAssignment/src/rahulM/Q4.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    static int  r;
+	public static void main(String [] a)
+	{
+	int l = Integer.parseInt(a[0]);
```
Comment (krishnakTechnocretids) [inline JavaAssignment/src/rahulM/Q4.java]: Give proper variable name.

File: JavaAssignment/src/rahulM/Q4.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+	int b = Integer.parseInt(a[1]);
+    int r = Integer.parseInt(a[2]);
+	rectangle(l,b);
+	circle(r);
+	}
+    static int rectangle(int l, int b)
```
Comment (krishnakTechnocretids) [inline JavaAssignment/src/rahulM/Q4.java]: What is rectangle ? Write method name like "areaOfRectangle"

**Pull request merged**